src/tests/unit/events/event_post_1.f90

```diff
@@ -0,0 +1,20 @@
+program event_post_1
+  use iso_fortran_env
+  implicit none
```

Comment:
Well the test tests event_post. Pure and easy. There is nothing more to it than whether it works or not. I liked the name, because it is just a feature test and with it the feature tested can be easily identified. What ideas for a better name do you have?